modules/EnsEMBL/Web/SpeciesDefs.pm

```diff
@@ -72,299 +73,39 @@ sub abbreviated_species_label {
 }
 ##
 
-#------------------------------------------------------------------------------
-# MULTI SPECIES
-#------------------------------------------------------------------------------
-
-sub _parse {
-  ### Does the actual parsing of .ini files
-  ### (1) Open up the DEFAULTS.ini file(s)
-  ### Foreach species open up all {species}.ini file(s)
-  ###  merge in content of defaults
-  ###  load data from db.packed file
-  ###  make other manipulations as required
-  ### Repeat for MULTI.ini
-  ### Returns: boolean
+sub _load_in_taxonomy_division {}
 
+sub _get_cow_defaults {
```

Comment:
this naming is going to make people confuse :-)

Reply:
Wasn't my name - copy-on-write is a bit of a mouthful! Maybe change cow to 'deepcopy'?

Reply:
Renamed (also done in ensembl-webcode)